egg.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Minor formatting of `json` file @michael-spengler

Reply:
thank you - just changed my template for future PRs with this.

Will do some more PRs to support nest.land ...